examples/skaffold/README.md

```diff
@@ -0,0 +1,138 @@
+# Multiple Cloud Functions, Same Host
```

Comment:
The H1 here and example content are not exactly what I would imagine.

Similar to https://github.com/GoogleCloudPlatform/functions-framework-python/issues/98#issuecomment-733872312, I wouldn't expect that the minimum requirement to run multiple functions on the same host involves Kubernetes/`minikube`/`skaffold`. I'd expect a lighter weight setup, like how the FF is very lightweight – maybe require 1 dependency like Docker.

---

That said, can we change this example / PR title to something more aligned to the content, like "Developing Multiple Functions using Minikube and Skaffold"